### PR TITLE
added new product page

### PR DESCRIPTION
**Pull request by piyapaymal2098**
i creat new product  page

### CODE REVIEW

File: ComcastCRMGUIFramework/src/test/java/practice/test/ProductPage.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package practice.test;
+
+public class ProductPage {
+	//step 1
+	//step2
```
Comment (ankitarevankar) [inline ComcastCRMGUIFramework/src/test/java/practice/test/ProductPage.java:5]: add the xpath for elements.

**Review by ankitarevankar — COMMENTED** [review]
add the comment

File: ComcastCRMGUIFramework/src/test/java/practice/test/ProductPage.java (hunk 1 above)
Comment (piyapaymal2098) [reply ComcastCRMGUIFramework/src/test/java/practice/test/ProductPage.java:5]: changes fixed

**Comment by piyapaymal2098** [comment]
done

**Comment by ankitarevankar** [comment]
project is merged

**Pull request merged**